Automattic/jetpack
# jetpack: Use `[contenthash]` in Webpack chunk names

#### Changes proposed in this Pull Request:

This will improve the long-term caching of the files by only busting
caches of the files that have changed rather than busting the cache of
every associated file when one does.

This should also help reduce the instances of the wpcom TeamCity build
deciding to change all the hashes, although completing that will also
require a fix for WordPress/gutenberg#34660.

#### Jetpack product discussion


p1631050115339300-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Check that the instant-search stuff and the extensions using swiper still work.
* Build locally, the hashes in the filenames should be the same regardless of the path you use.
* Change an instant-search CSS file and rebuild, only the built CSS file's hash should change.
